code/game/verbs/ooc.dm

```diff
@@ -97,8 +97,8 @@
 		if(!looc_allowed)
 			to_chat(src, SPAN_DANGER("LOOC is globally muted"))
 			return
-		if(!dlooc_allowed && (mob.stat == DEAD || isobserver(mob)))
-			to_chat(usr, SPAN_DANGER("LOOC for dead mobs has been turned off."))
+		if(!dlooc_allowed && (mob.stat != CONSCIOUS || isobserver(mob)))
```

Comment:
you could use just if(mob.stat) 😈  because i think concious was 0 and dead + inconcious was 1 and 2